Sources/KIF/UIAutomationHelper.m

```diff
@@ -213,7 +214,12 @@ - (BOOL)_alertIsValidAndVisible:(UIAAlert *)alert;
 }
 
 - (void)linkAutomationFramework {
-    dlopen([@"/Developer/Library/PrivateFrameworks/UIAutomation.framework/UIAutomation" fileSystemRepresentation], RTLD_LOCAL);
+    NSString *path = @"/Developer/Library/PrivateFrameworks/UIAutomation.framework/UIAutomation";
```

Comment:
Does this help anything for uiautomation?

Reply:
tried to find why UIAutomation fails to load, see that `loadDylibForSimulator` prepends IPHONE_SIMULATOR_ROOT to framework path, but it didn't helped, reverted in https://github.com/openium/KIF/commit/4f286cf6f0264c1b93b5b1f860f00ca0bfb43104